README.md

```diff
@@ -1,80 +1,74 @@
 
 To get a sample app up and running in one command, run the following:
 
 ```sh
-git clone git@github.com:aws-samples/amazon-ecs-cli-sample-app.git demo-app
-cd demo-app
-ecs-preview init --project demo      \
-  --app api                          \
-  --app-type 'Load Balanced Web App' \
-  --dockerfile './Dockerfile'        \
-  --port 80                          \
+$ git clone git@github.com:aws-samples/amazon-ecs-cli-sample-app.git demo-app
+$ cd demo-app
+$ copilot init --app demo                \
+  --svc api                              \
+  --svc-type 'Load Balanced Web Service' \
+  --dockerfile './Dockerfile'            \
+  --port 80                              \
   --deploy
 ```
 
 This will create a VPC, Application Load Balancer, an Amazon ECS Service with the sample app running on AWS Fargate. This process will take around 8 minutes to complete - at which point you'll get a URL for your sample app running!
 
-<details>
-    <summary> watch what happens when you run <tt>init</tt></summary>
-
-![Step By Step Setup](https://user-images.githubusercontent.com/828419/69770895-91813f80-113f-11ea-8be9-60df6c2bf3fc.gif)
-</details>
-
-### Cleaning up 🧹
+## Cleaning up 🧹
 
-Once you're finished playing around with this project, you can delete it and all the AWS resources associated it by running `ecs-preview project delete`.
+Once you're finished playing around with this project, you can delete it and all the AWS resources associated it by running `copilot app delete`.
 
-### Learning more 📖
+## Learning more 📖
 
-Want to learn more about what's happening? Check out our [getting started guide](https://github.com/aws/amazon-ecs-cli-v2/wiki/Getting-Started) and a detailed breakdown of all the [infrastructure](https://github.com/aws/amazon-ecs-cli-v2/wiki/Infrastructure) that gets created.
+Want to learn more about what's happening? Check out our documentation [https://aws.github.io/copilot-cli/](https://aws.github.io/copilot-cli/) for a getting started guide, learning about Copilot concepts, and a breakdown of our commands. 
 
 ## We need your feedback 🙏
 
-The ECS CLI v2 is in developer preview, meaning we want to know what works, what doesn't work, and what you want! Have any feedback at all? Drop us an [issue](https://github.com/aws/amazon-ecs-cli-v2/issues/new) or join us on [gitter](https://gitter.im/aws/amazon-ecs-cli-v2?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).
+The AWS Copilot CLI is in beta, meaning that you can expect our command names to be stable as well as the shape of our 
+infrastructure patterns. However, we want to know what works, what doesn't work, and what you want! 
+Have any feedback at all? Drop us an [issue](https://github.com/aws/copilot-cli/issues/new) or join us on [gitter](https://gitter.im/aws/amazon-ecs-cli-v2?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).
```

Comment:
```suggestion
Have any feedback at all? Drop us an [issue](https://github.com/aws/copilot-cli/issues/new) or join us on [gitter](https://gitter.im/aws/copilot-cli?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).
```

Reply:
Updated also the badge at the top of the README!